src/components/DragCaptureZone/DragCaptureZone.spec.jsx

```diff
@@ -0,0 +1,160 @@
+import _ from 'lodash';
+import assert from 'assert';
+import React from 'react';
+import SyntheticEvent from 'react/lib/SyntheticEvent';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import describeWithDOM from '../../util/describe-with-dom';
+import { common } from '../../util/generic-tests';
+
+import DragCaptureZone from './DragCaptureZone';
+
+describe('DragCaptureZone', () => {
+	common(DragCaptureZone);
+
+	describe('props', () => {
+		describe('onDrag', () => {
+			it('defaults to the Lodash `noop` method.', () => {
+				const wrapper = mount(<DragCaptureZone />);
+
+				assert.equal(wrapper.prop('onDrag'), _.noop);
+			});
+		});
+
+		describe('onDragEnd', () => {
+			it('defaults to the Lodash `noop` method.', () => {
+				const wrapper = mount(<DragCaptureZone />);
+
+				assert.equal(wrapper.prop('onDragEnd'), _.noop);
+			});
+		});
+
+		describe('onDragStart', () => {
+			it('defaults to the Lodash `noop` method.', () => {
+				const wrapper = mount(<DragCaptureZone />);
+
+				assert.equal(wrapper.prop('onDragStart'), _.noop);
+			});
+		});
+
+		describe('pass throughs', () => {
+			it('passes through all props to the root element.', () => {
+				const wrapper = mount(<DragCaptureZone foo={1} bar={2} baz={3} />);
+				const rootProps = wrapper.find('div').props();
+
+				_.forEach(['foo', 'bar', 'baz'], (prop) => {
+					assert(_.has(rootProps, prop));
+				});
+			});
+		});
+	});
+});
+
+describeWithDOM('DragCaptureZone', () => {
+	const dragCoordinates = {
+		pageX: 75,
+		pageY: 25
+	};
+	const dragEndCoordinates = {
+		pageX: 25,
+		pageY: 50
+	};
+	const dragStartCoordinates = {
+		pageX: 50,
+		pageY: 75
+	};
+
+	describe('user presses the mouse button down over the rendered component', () => {
+		let onDragStart;
+
+		beforeEach(() => {
+			onDragStart = sinon.spy();
+
+			mount(<DragCaptureZone onDragStart={onDragStart} />).find('div').simulate('mousedown', dragStartCoordinates);
+		});
+
+		it('calls the function passed in as the `onDragStart` prop...', () => {
+			assert(onDragStart.calledOnce);
+		});
+
+		it('...passes along a map of coordinate data as the first argument...', () => {
```

Comment:
I do that mainly for readability for whoever is consuming the tests where I feel that there's a viable separation of tests. I don't feel _super_ strongly about combining these together, though.